### ISSUE TITLE
Failed to add chart repository with helm when OIDC is enabled

### ISSUE THREAD

**Issue by ywk253100** [goharbor/harbor#7625]
```
Apr 30 02:48:34 192.168.16.1 core[20741]: 2019-04-30T09:48:34Z [ERROR] [security.go:435]: failed to authenticate test3: Unrecognized auth_mode: oidc_auth
Apr 30 02:48:34 192.168.16.1 core[20741]: 2019-04-30T09:48:34Z [ERROR] [base.go:69]: GET /chartrepo/test3/index.yaml failed with error: {"code":401,"message":"Unauthorized"}
```

**Comment by reasonerjt** [comment]
Fixed by #7645

**Comment by loosi** [comment]
Is this issue still a thing? 
Using 1.8.1 with OIDC and still getting the same error with OIDC and helm push. (though docker login works with the same cli secret)

```
echo 4b01foduqa..... | docker login --username mru --password-stdin https://registry.company
Login Succeeded
```
```
helm push redmine-9.1.0.tgz https://mr:4b01foduqa.....@registry.company/chartrepo/library
Error: 401: could not properly parse response JSON: {"code":401,"message":"Unauthorized"}
[ERROR] [/core/filter/security.go:471]: failed to authenticate mru: Unrecognized auth_mode: oidc_auth
[ERROR] [/common/api/base.go:69]: POST /api/chartrepo/library/charts failed with error: {"code":401,"message":"Unauthorized"}
```

Anything else to do with Helm push here? I've tried using helm push following the manual (https://github.com/goharbor/harbor/blob/master/docs/user_guide.md#working-with-helm-cli), but no matter what method : always 401
